### DISCUSSION TITLE
Questions regarding navigations and cascadings

### DISCUSSION THREAD

**Discussion by alexmurari** [nhibernate/nhibernate-core#2686] [Q&A]
Hi,

I have two questions, searched everything but couldn't find anything:

1. There's a way to explicit load (fetch) navigations of an already loaded entity?
Example: `EntityEntry(myEntityObj).Fetch(t => t.SomeProperty);`

2. There's a way to atomically set the cascade behavior at a query level, to prevent an Update operation from persisting the related entities/navigations?
Example: `session.Update(myEntityObj).IgnoreRelated();` or like EF `ctx.Entry(entity).State = EntityState.Modified;`

Thanks!

**Comment by fredericDelaporte** [comment]
1- you are looking for `NHibernateUtil.Initialize(myEntityObj.SomeProperty)`, or its async counterpart.
2- I do not know if there is something for this.
